currency/manager.go

```diff
@@ -208,8 +205,7 @@ func (p *PairsManager) StorePairs(a asset.Item, pairs Pairs, enabled bool) error
 
 	// NOTE: Length check not needed in this scenario as it has the ability to
 	// remove the entire stored list if needed.
-	cpy := make(Pairs, len(pairs))
-	copy(cpy, pairs)
+	cpy := slices.Clone(pairs)
```

Comment:
suggestion: can rm the var and comment and inline to assignment.

Reply:
Yep. In doing so I'm going to remove the comment, which needed rewording at best, but feels implicit to me.

Fixed ef4e43c